region/lowernorfair/east.json

```diff
@@ -1640,6 +1693,16 @@
           "id": "B",
           "name": "Bomb Blocks of Shame",
           "obstacleType": "inanimate"
+        },
+        {
+          "id": "C",
```

Comment:
This is a complex room. I think we'd better update the image with the new obstacles.

Reply:
Yes, another thing I forgot to do.